### ISSUE TITLE
include:: output macro does not always work in 'current' docs

### ISSUE THREAD

**Issue by lonlylocly** [gradle/gradle#13709]
Hello,

In "current" docs (6.5.1) page like this one:

https://docs.gradle.org/current/userguide/tutorial_using_tasks.html

I expect to see output of samples, but I see the raw code with "include" instead, like this:

```
Output of gradle -q upper
> gradle -q upper
include::{snippetsPath}/tutorial/upper/tests/upper.out
```
![2020-07-07_11-59-24](https://user-images.githubusercontent.com/7804425/86765850-00b20100-c04a-11ea-9b72-9404afddcaf4.png)

Happens also in other pages, like this one: https://docs.gradle.org/current/userguide/build_environment.html

Does NOT happen in version of docs 6.5: https://docs.gradle.org/6.5/userguide/tutorial_using_tasks.html

P.S. I am not sure if this is the right place to report this, in this case please provide instructions of where I should report this.

### Expected Behavior

See the content of `tutorial/upper/tests/upper.out` under `Output of...`

### Current Behavior

See this instead 

```
include::{snippetsPath}/tutorial/upper/tests/upper.out
```

### Context

I am trying to learn Gradle and these includes are really confusing me. It took me several days to realize that something's fishy, and this is not the intended output of gradle.

It made my learning process harder.

### Steps to Reproduce 

Open this page in a browser: 
https://docs.gradle.org/current/userguide/tutorial_using_tasks.html

### Your Environment

Google Chrome Version 83.0.4103.116 (Official Build) (64-bit)

**Comment by big-guy** [comment]
This was a bug in our documentation generation and was fixed on master, but not on the 6.5.1 branch at the time.  We redeployed some fixed docs.

Thanks!

**Comment by lonlylocly** [comment]
🙏 Thank you!
